core/services/pipeline/task.jsonparse_test.go

```diff
@@ -192,7 +191,7 @@ func TestJSONParseTask(t *testing.T) {
 			"false",
 			pipeline.NewVarsFrom(nil),
 			[]pipeline.Result{{Value: `{"data": [[0, 1]]}`}},
-			[]interface{}{json.Number("0"), json.Number("1")},
+			[]interface{}{int64(0), int64(1)},
```

Comment:
Do we perhaps also want to keep the previous case?

Reply:
After json unmarshalling and processing all `json.Number`s there should be no `json.Number`s left in a tree. We should always convert those to numbers, or fail.